acceptance/setup/aio/pre-suite/010_Install.rb

```diff
@@ -7,31 +7,46 @@
 step "Install repositories on target machines..." do
 
   sha = ENV['SHA']
-  server_version = ENV['SERVER_VERSION'] ||= 'nightly'
   repo_configs_dir = 'repo-configs'
 
   hosts.each do |host|
     install_repos_on(host, 'puppet-agent', sha, repo_configs_dir)
   end
 
-  install_repos_on(master, 'puppetserver', server_version, repo_configs_dir)
+  if master['passenger']
+    passenger_version = ENV['PASSENGER_VERSION'] ||= '940f74a0b0c6ef1bbdbaf6bd9fd9c3ed2a6b981c'
```

Comment:
nitpicky question: why use `||=` here instead of `||` (same question below)? is setting ENV['PASSENGER_VERSION'] desirable?